.devcontainer/Dockerfile

```diff
@@ -310,14 +312,44 @@ RUN git clone https://github.com/OPENAIRINTERFACE/opencord.org.freeDiameter.git
     make -j"$(nproc)" && \
     make install && \
     cd / && \
-    rm -rf freediameter
+    rm -rf freediameter \
+    rm -rf /tmp/*
 
 ##### Go language server support for vscode
 RUN go get -v golang.org/x/tools/gopls
 
 #### Update shared library configuration
 RUN ldconfig -v
 
+
+##### Install Python requirements
+RUN pip3 install --quiet --upgrade --no-cache-dir "setuptools==49.6.0"
+
+### install python3-aioeventlet from the magma apt repo (from .circleci/config.yml)
+COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub ${MAGMA_ROOT}/orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub
+RUN cat ${MAGMA_ROOT}/orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub | apt-key add - \
+    && echo "deb https://facebookconnectivity.jfrog.io/artifactory/list/dev-focal/ focal main" | tee /etc/apt/sources.list.d/fbc.list \
+    && apt-get update -y  \
+    && apt-get install -y python3-aioeventlet \
+    && rm -rf /var/lib/apt/lists/*  # delete lists that are downloaded by apt-get update
+
+# add patch that is missing in jfrog version of aioeventlet (it only comes with 1 of 2 patches)
+COPY lte/gateway/deploy/roles/magma/files/patches/aioeventlet_fd_exception.patch /tmp/
+RUN patch -N -s -f /usr/local/lib/python3.8/dist-packages/aioeventlet.py < /tmp/aioeventlet_fd_exception.patch \
+    && rm -rf /tmp/*
```

Comment:
There are two patches for aioeventlet in `lte/gateway/deploy/roles/magma/files/patches`. The jfrog aioeventlet package only contains one of them. Here we add the second one.

Reply:
Should we update the one in jfrog to have both of the patches? Is that easy to do? (cc @tmdzk / @quentinDERORY )

Reply:
It is rather hard to find, how the custom deb-packages for aioeventlet were provided. If this was done with [the buildscript in the third party folder](https://github.com/magma/magma/blob/0fb795e1fdd2d01e8dff9dcb9b2123208c0874d2/third_party/build/bin/aioeventlet_build.sh#L44) I would have expected that all patches from [the respective folder were] applied(https://github.com/magma/magma/tree/master/third_party/build/patches/aioeventlet).
(Side comment: Afaiks devops privileges are required to upload to https://facebookconnectivity.jfrog.io anyway.)

In any way, I agree that aioeventlet is a big pain and the efforts to improve should be taken some-when. Those efforts are tracked in #6368. While removal of aioeventlet might be desirable, it is not easy. However, also meaningful improvements to the current situation would be bigger topics (https://github.com/magma/magma/issues/6368#issuecomment-1025645600), which I would see clearly out of scope of this PR and effort.

Our goal here was to quickly improve the devcontainer support for Python with low effort. By staying rather close to the current Makefile setup we try bring missing functionality to the devcontainer, following the approach "First make it work, than make it pretty."

Reply:
Thanks for looking into this! Yes, I agree with make it work then make it pretty, as long as we are filing issues to track the potential clean up opportunities (which you are doing, thanks!).

Maybe we can sync with @pshelar if he knows how aioeventlet patches are handled.